src/artemis/devices/I03Smargon.py

```diff
@@ -0,0 +1,25 @@
+from ophyd import Component as Cpt
+from ophyd import Device, EpicsMotor, EpicsSignal
+
+"""
+Real motors added to allow stops following pin load (e.g. real_x1.stop() )
+Stub offsets are calibration values that are required to move between calibration pin position and spine pins. These are set in EPCICS and applied via the proc.
+"""
+
+
+class I03Smargon(Device):
```

Comment:
Should: There is already an I03 smargon in `devices/motors.py`. Could you add any additionally required PVs there? Or if you think it makes sense to have it in a separate smargon file we can do that too. Just don't want two copies

Reply:
I think there is a discussion to be had on devices and how granular you want them in terms of file structure. Motors.py is either going to become very large, or should be split to individual devices. Should something similar to the EPICS grouping be used?

Reply:
Yh, I agree that motors was probably too large a category. It made sense when the usecase was just grid scan but now we're looking wider afield it doesn't make as much sense. Happy to move it to here. In which case can you merge the one in `motors.py` and change all references? I can do this if you'd rather.

Reply:
I will merge this PR and fix this in a subsequent issue